icon.svg

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Consider making it part of your code repository as it's not specific to Flathub.